packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -1,11 +1,30 @@
+// @ts-checck
```

Comment:
We might consider renaming this - it's meant to capture the `OrchestrationAccount` interface implemented for a `LocalChainAccount`.